_posts/en/newsletters/2023-08-23-newsletter.md

```diff
@@ -0,0 +1,245 @@
+    storage][topic peer storage] described in [BOLTs #881][], the
+    version experimentally implemented in Core Lightning earlier this
+    year (see [Newsletter #238][news238 peer storage]), and (according
+    to a message from Bastien Teinturier [indicates][teinturier
+    backups]) a verson of the scheme implemented in the Phoenix wallet
+    for LN.
+
+
+  <!-- I've previously confirmed that "ghost43" (all lowercase) is how
+  they'd like to be attributed -->
+  - A [reply][ghost43 backups] by ghost43 explained that fraud proofs
+    leading to financial penalties is a powerful tool for a client
+    storing data with anonymous peers.  A large popular service might
+    care about its reputation enough to avoid lying to its clients, but
+    anonymous peers don't have any reputation to lose.  Also suggested
+    by ghost43 was a modification of the protocol to make it symmetric,
+    so in addition to Alice storing her state with Bob (and Bob being
+    penalized for lying), Bob could store his state with Alice and she
+    could be penalized for lying.
+
+      Voegtlin extended this idea to [warn][voegtlin backups2] that the
+      providers of wallet software have a significant need for a good
+      reputation and that they lose reputation when their users lose
+      funds---even if the software functioned as best as possible.  As
+      the developer of wallet software, it is thus very important to him
+      to minimize the risk of an anonymous peer being able to steal from
+      an Electrum user who uses a mechanism like peer backups.
+
+  There was no clear resolution to the discussion.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Scaling Lightning call for feedback:**
+  [Scaling Lightning][] is a testing toolkit for the Lightning Network on
+  regtest and signet. The project aims to provide tools to test different LN
+  implementations in a variety of configurations and scenarios. The project
+  provided a recent [video update][sl twitter update] for the community. LN
+  developers, researchers, and infrastructure operators are encouraged to
+  [provide feedback][sl tg].
+
+- **Torq v1.0 released:**
+  [Torq][torq github], an LN node management software, [announced][torq blog] a
+  v1.0 release including Lightning Service Provider (LSP) features, automation
+  workflows, and advanced features for large node operators.
+
+- **Blixt Wallet v0.6.8 released:**
+  The [v0.6.8 release][blixt v0.6.8] includes support for [hold invoices][topic
+  hold invoices] and [zero-conf channels][topic zero-conf channels] among other improvements.
+
+- **Sparrow 1.7.8 released:**
+  Sparrow [1.7.8][sparrow 1.7.8] added support for [BIP322][] [message signing][topic generic
+  signmessage], including P2TR addresses, and made various improvements to
+  [RBF][topic rbf] and [CPFP][topic cpfp] fee bumping features.
+
```

Comment:
```suggestion

- **Wasabi Wallet 2.0.4 released:**
  Wasabi [2.0.4][wasabi 2.0.4] includes a daemon, TurboSync (for faster wallet load), 
  privacy suggestions when sending, a _Speed Up Transaction_ button for increasing the
  fee of an unconfirmed transaction with [RBF][topic rbf] or [CPFP][topic cpfp], a new 
  amount decomposer for the coinjoins and lots of other improvements and bug fixes.
```

Reply:
@MarnixCroes thanks for the heads up. Reworded a bit and added here: https://github.com/bitcoinops/bitcoinops.github.io/pull/1250/commits/c3d8c4a97f756b0bd506dbda49c66103bc9c6a39